ODLA/platforms/dnnl/odla_dnnl.cc

```diff
@@ -1261,23 +1336,17 @@ odla_value odla_BatchNormalization(odla_value input,
                {DNNL_ARG_DST, weight_mem}});
   }
   auto op_desc = dnnl::batch_normalization_forward::desc(
-      dnnl::prop_kind::forward, input_md, epsilon, flags);
+      dnnl::prop_kind::forward_inference, input_md, epsilon, flags);
```

Comment:
just wondering: what's the difference between "forward" & "forward_inference" ?

Reply:
I'm not too sure either..